src/cca/islet_test/Makefile

```diff
@@ -1,27 +1,94 @@
+# #############################################################################
+# File: src/cca/islet_test/Makefile
+# #############################################################################
+
+CC=g++
 GPP=g++
+PROTO=protoc
+
+# Symbol to CERTIFIER_ROOT/src/ dir
+ifndef SRC_DIR
+SRC_DIR=../..
+endif
+
+ifndef OBJ_DIR
+OBJ_DIR=.
+endif
+
+ifndef LOCAL_LIB
+LOCAL_LIB=/usr/local/lib
+endif
+
+S = $(SRC_DIR)
+O = $(OBJ_DIR)
+CCAS = $(S)/cca
+US = .
+
+I = $(SRC_DIR)/../include
 ISLET_PATH = ../../../third_party/islet
-ISLET_INCLUDE= -I$(ISLET_PATH)/include
-ISLET_LDFLAGS= -L$(ISLET_PATH)/remote/out/x86_64-unknown-linux-gnu/debug/ -lislet_sdk
+ISLET_INCLUDE= -I $(ISLET_PATH)/include
+ISLET_LDFLAGS= -L $(ISLET_PATH)/lib -lislet_sdk
+
+CERTIFIER_INCLUDE= -I ../../../include
+
+# List of dependent objects
+dobj = $(O)/certifier.pb.o $(O)/shim_test.o $(O)/cca_shim.o $(O)/support.o
 
 .PHONY: all
 all: attest_seal_test
 
-CFLAGS += $(ISLET_INCLUDE)
-LDFLAGS += $(ISLET_LDFLAGS)
+# Compilation of protobuf files could run into some errors, so avoid using
+# # -Werror for those targets
+CFLAGS_NOERROR = -I $(I) -O3 -g -Wall -std=c++11 -Wno-unused-variable -D X64 -Wno-deprecated-declarations -D CCA_CERTIFIER
+CFLAGS += $(CFLAGS_NOERROR) -Werror $(ISLET_INCLUDE) $(CERTIFIER_INCLUDE)
+
+LDFLAGS = $(ISLET_LDFLAGS) -L $(LOCAL_LIB) -lprotobuf -L /usr/local/opt/openssl@1.1/lib/ -lcrypto -lssl
 
 .PHONY: attest_seal_test
 attest_seal_test: attest_seal_test.cc
+	@echo "Compiling $<"
 	@$(GPP) $< $(CFLAGS) $(LDFLAGS) -o $@
-	@LD_LIBRARY_PATH=$(ISLET_PATH)/remote/out/x86_64-unknown-linux-gnu/debug ./$@
+	LD_LIBRARY_PATH=$(ISLET_PATH)/lib ./$@
```

Comment:
This change came in from a PR @yelvmw worked on, and commit SHA to `/main` is `8d2d255`.

But this change was found to be not necessary while developing this CCA-shim support. Things worked just fine with `/lib` on Nimbus-VM and also on CI-build machines.